src/simulators/qasm/qasm_controller.hpp

```diff
@@ -340,6 +347,8 @@ void QasmController::set_config(const json_t &config) {
   JSON::get_value(extended_stabilizer_measure_sampling_,
                   "extended_stabilizer_measure_sampling", config);
 
+  JSON::get_value(use_GPUs_,"GPU", config);
```

Comment:
Instead of checking for `"GPI"` in `backend_options` can we have it as an explicit method `"method": "statevector_gpu`

Reply:
I added "method": "statevector_gpu"